vmm/src/memory_manager.rs

```diff
@@ -1479,6 +1481,7 @@ impl MemoryManager {
             zone_id: DEFAULT_MEMORY_ZONE.to_string(),
             virtio_mem: false,
             file_offset: 0,
+            userspace_addr: region.as_ptr() as u64,
```

Comment:
I think it's called `host_addr` in other part of the code and in the `vm-memory` crate.

Reply:
Ack I will update it to host addr.